core/models/volumes.go

```diff
@@ -6,7 +6,10 @@ type Volume struct {
 	ID string `json:"id"`
 	// IsRoot specifies that the volume is to be used as the root volume. A machine
 	// must have a root volume.
-	IsRoot bool `json:"is_root"`
+	//
+	// NOTE: This field does not exist on the gRPC API, it is automatically filled
+	// when converting from the API model to the this internal model.
+	// IsRoot bool `json:"is_root"`
```

Comment:
Is this comment `//` intentional on the Volume model? This is the internal model. Either the comment on the field or the commented out field seems odd.

Reply:
The comment _was_ valid, but I removed the need for `is_root` completely and forget to remove this altogether.